### PR TITLE
Add vm target to pegen script to generate the vm parser

### PR DESCRIPTION
**Pull request by lysnikolaou**

### CODE REVIEW

File: Tools/peg_generator/Makefile [hunk 1]
```diff
@@ -25,6 +25,9 @@ build: peg_extension/parse.c
 peg_extension/parse.c: $(GRAMMAR) $(TOKENS) pegen/*.py peg_extension/peg_extension.c ../../Parser/pegen/pegen.c ../../Parser/pegen/parse_string.c ../../Parser/pegen/*.h pegen/grammar_parser.py
 	$(PYTHON) -m pegen -q c $(GRAMMAR) $(TOKENS) -o peg_extension/parse.c --compile-extension
 
+generate_vm: $(GRAMMAR) $(TOKENS) pegen/*.py ../../Parser/pegen/pegen.c ../../Parser/pegen/parse_string.c ../../Parser/pegen/*.h
+	$(PYTHON) -m pegen -q c $(GRAMMAR) $(TOKENS) -o peg_extension/vmparse.h --vm
```
Comment (gvanrossum) [inline Tools/peg_generator/Makefile:29]: I've been generating directly to ../../Parser/pegen/vmparse.h.

File: Tools/peg_generator/pegen/vm_generator.py [hunk 2]
```diff
@@ -90,8 +94,8 @@ def visit_StringLeaf(self, node: StringLeaf) -> Tuple[str, str]:
                 return self.keyword_helper(val)
             else:
                 return self.soft_keyword_helper(val)
-        tok_num: int = token.EXACT_TOKEN_TYPES[val]  # type: ignore [attr-defined]
-        return "OP_TOKEN", token.tok_name[tok_num]
+        tok_num: int = self.exact_tokens[val]  # type: ignore [attr-defined]
```
Comment (gvanrossum) [inline Tools/peg_generator/pegen/vm_generator.py:97]: You don't need the `type: ignore` comment any more.

**Comment by lysnikolaou** [comment]
> Huh, somehow I had thought this would become a separate subcommand, like `c` and `python`. But this works too.

If you think that this would be better, I can certainly do it.

Pushed some new changes that address your review comments.

**Comment by pablogsal** [comment]
> > Huh, somehow I had thought this would become a separate subcommand, like `c` and `python`. But this works too.

Yeah, I would also prefer that this is a target at the same level as `c` and `python` to keep it consistent.

**Comment by lysnikolaou** [comment]
Since you both agree that a `vm` target is better, I replaced the `--vm` flag with a `vm` target.

**Pull request merged**